.gitignore

```diff
@@ -7,3 +7,4 @@ web-ext-artifacts/
 test/dist/*
 ckeditor-build/build/*
 build/
+.DS_Store
```

Comment:
Please add this to your global git ignore on your machine, not this file

Reply:
Done! should have been done a long time ago, thanks